docs/MAC_SETUP.md

```diff
@@ -0,0 +1,103 @@
+# Mac Development Setup
+
+## Application Installs
+
+* Install [Homebrew](https://brew.sh/) this will prompt for a bunch of passwords entries. It is working just get to the end!
+* Install [Docker Desktop](https://docs.docker.com/desktop/install/mac-install/)
+* Install [VSCode](https://formulae.brew.sh/cask/visual-studio-code)
+    * Install [Dev Containers](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extension
+
+## GitHub Setup
+
+Create an ssh keypair:
+
+```
+ssh-keygen -t rsa -a 100 -Z aes128-gcm@openssh.com
+```
+
+Create a file `config` in the ~/.ssh folder:
+
+```
+touch ~/.ssh/config
+```
+
+Add the following to the file:
+
+```
+Host github.com
+    AddKeysToAgent yes
+    User git
+    PubkeyAcceptedAlgorithms +ssh-rsa
+```
+
+## Standup Dev Playground
+
+### TL;DR
+
+This repo includes a script that does the setup steps below checking that your environment is setup correctly 
+and if so running the commands to stand up the Docker network and dev playground.
+
+```shell
+$> bin/devup
+```
+
+### Create docker network
+
+```
+docker network create --driver bridge --ipv6  --subnet fd00::/80 infranet_network --attachable
+```
+
```

Comment:
I generally don't like replicating the steps from a script in the README because then there is not a single source of truth, and we have to ensure that the two are in sync.

Having said that, maybe this is just my resource-strapped open source project experience speaking, and better resources projects will devote the time to keeping them in sync.

Although this is also an open-source project... 😄

Reply:
I'm actually in agreement with you on this one. My preference is to have any of these in a scripts directory on path to be referenced from the README. My worry is that this isn't the current EVerest way of doing things, and that maybe it's better that I just remove the scripts altogether, no matter how convenient I find them.

Reply:
I've removed the script so that there's no confusion.